### PR TITLE
Fix typo in install the contao core bundle step

### PR DESCRIPTION
**Pull request by bytehead**
See #527

### CODE REVIEW

**Review by fritzmg — APPROVED** [review]

File: docs/dev/getting-started/initial-setup/symfony-application/contao-4.9.md [hunk 1]
```diff
@@ -43,15 +43,18 @@ you can proceed to the second step, the installation of Contao itself.
 
 ```
 $ composer require \
+    doctrine/common:^2.1 \
     doctrine/dbal:^2.10 \
     doctrine/doctrine-bundle ^1.8 \
```
Comment (fritzmg) [inline docs/dev/getting-started/initial-setup/symfony-application/contao-4.9.md]:
```suggestion
    doctrine/doctrine-bundle:^1.8 \
```

**Review by fritzmg — CHANGES_REQUESTED** [review]

File: docs/dev/getting-started/initial-setup/symfony-application/contao-4.9.md (hunk 1 above)
Comment (bytehead) [reply docs/dev/getting-started/initial-setup/symfony-application/contao-4.9.md]: 🙄

**Review by fritzmg — APPROVED** [review]

**Comment by fritzmg** [comment]
Thank you @bytehead  :)

But is that all? @anniahoi seemed to have some problems with missing services as well.

**Comment by bytehead** [comment]
Not yet.

**Pull request merged**